src/main/java/roomescape/infrastructure/LoginMemberArgumentResolver.java

```diff
@@ -0,0 +1,41 @@
+package roomescape.infrastructure;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.core.MethodParameter;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+import roomescape.member.LoginMember;
+import roomescape.member.Member;
+import roomescape.member.MemberService;
+
+@Component
+public class LoginMemberArgumentResolver implements HandlerMethodArgumentResolver {
+    private MemberService memberService;
+    private JwtTokenUtil jwtTokenUtil;
+
+    public LoginMemberArgumentResolver(MemberService memberService, JwtTokenUtil jwtTokenUtil) {
+        this.memberService = memberService;
+        this.jwtTokenUtil = jwtTokenUtil;
+    }
+
+    @Override
+    public boolean supportsParameter(MethodParameter parameter) {
+        return parameter.getParameterType().equals(LoginMember.class);
+    }
+
+    @Override
+    public Object resolveArgument(MethodParameter parameter,
+                                  ModelAndViewContainer mavContainer,
+                                  NativeWebRequest webRequest,
+                                  WebDataBinderFactory binderFactory) throws Exception {
+        HttpServletRequest httpServletRequest = (HttpServletRequest) webRequest.getNativeRequest();
+
+        Long memberId = jwtTokenUtil.getPayload(httpServletRequest);
+        Member member = memberService.findMemberById(memberId);
```

Comment:
토큰이 없거나 회원 정보를 찾을 수 없는 경우 예외 처리를 해주면 좋을 거 같아요!
```suggestion
        Long memberId = jwtTokenUtil.getPayload(httpServletRequest);
        if (memberId == null) {
            throw new IllegalArgumentException("Invalid or missing JWT token");
        }

        Member member = memberService.findMemberById(memberId);
        if (member == null) {
            throw new IllegalArgumentException("Member not found for id: " + memberId);
        }
```

Reply:
예외 처리 진행했습니다!